GitUI/Theme/ControlUtils.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Collections.Generic;
+using System.Windows.Forms;
+
+namespace GitUI
+{
+    internal static class ControlUtils
```

Comment:
This class contains extensions, and it should be named accordingly
```suggestion
    internal static class ControlTagExtensions`
```

The class also feels misplaced in the "Theme" folder.
A number of extensions live in GitExtUtils/GitUI, perhaps it is more appropriately placed there

Reply:
ok